### PR TITLE
Prepare release 2.0.0

### PR DESCRIPTION
**Pull request by abikouo**
##### SUMMARY
- Run `antsibull-changelog` to generate release notes.
- Update `galaxy.yml` version to `2.0.0`

##### ISSUE TYPE
- Release Pull Request

### CODE REVIEW

**Review by hakbailey — APPROVED** [review]

File: changelogs/changelog.yaml [hunk 1]
```diff
@@ -23,3 +23,142 @@ releases:
...
+    - aws_setup_credentials_add_no_log.yml
+    - awsconfig_apigateway.yml
+    - create_rds_global_cluster_role.yml
+    - customized_ami_doc_update.yaml
+    - fix_manage_vpc_peering_integration_test.yml
+    - fix_sanity_issues.yaml
+    - transit_gateway_doc_update.yaml
+    - update_ansible_lint_action.yml
+    - webapp_in_HA.yaml
+    release_date: '2023-11-15'
```
Comment (GomathiselviS) [inline changelogs/changelog.yaml:164]: We might have to change this based on when we release. I hope that will be take care of.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -5,6 +5,70 @@ CHANGE THIS IN changelogs/config.yaml! Release Notes
 .. contents:: Topics
 
 
+v2.0.0
+======
+
```
Comment (alinabuzachis) [inline CHANGELOG.rst:10]: Can you please add a release_summary?

File: CHANGELOG.rst [hunk 3]
```diff
@@ -5,6 +5,70 @@ CHANGE THIS IN changelogs/config.yaml! Release Notes
...
+Minor Changes
+-------------
+
+- Add a playbook to deploy a simple flask web app into high availability architecture (https://github.com/redhat-cop/cloud.aws_ops/pull/97).
```
Comment (alinabuzachis) [inline CHANGELOG.rst:14]: I guess some new roles are not shown in the changelog. I don't know if there's a way to automatically add then at changelog generation, but if not, we should manually create a changelog for them.

Comment (abikouo) [reply CHANGELOG.rst:19]: done

File: CHANGELOG.rst [hunk 4]
```diff
@@ -5,6 +5,90 @@ CHANGE THIS IN changelogs/config.yaml! Release Notes
...
+
+Release Summary
+---------------
+
+This release fixes bug on roles, remove support for ansible-core < 2.14 and introduces new features.
+
+Minor Changes
+-------------
+
+- Add a playbook to deploy a simple flask web app into high availability architecture (https://github.com/redhat-cop/cloud.aws_ops/pull/97).
```
[the remark this answers is not in the document]
Comment (abikouo) [reply CHANGELOG.rst:19]: yes it makes sense

**Review by GomathiselviS — APPROVED** [review]

**Pull request merged**